### PR TITLE
Added mini boiler configs

### PR DESCRIPTION
**Pull request by Kurtchekov**

### CODE REVIEW

**Comment by DaedalusGame** [comment]
>mini boiler can explode config

casuls everywhere

File: src/main/java/teamroots/embers/recipe/RecipeRegistry.java [hunk 1]
```diff
@@ -260,7 +260,7 @@ public void initEarly(RegistryEvent.Register<IRecipe> event) {
 		EmbersAPI.registerCatalysisFuel(new OreIngredient("gunpowder"),3.0);
 		EmbersAPI.registerCatalysisFuel(new OreIngredient("dustGlowstone"),4.0);
 
-		EmbersAPI.registerBoilerFluid(FluidRegistry.WATER,FluidRegistry.getFluid("steam"),5.0);
+		EmbersAPI.registerBoilerFluid(FluidRegistry.WATER,FluidRegistry.getFluid("steam"),ConfigManager.miniBoilerWaterMultiplier);
```
Comment (DaedalusGame) [inline src/main/java/teamroots/embers/recipe/RecipeRegistry.java]: this should really be crafttweaker integration. If you don't want to touch that, could you remove this change for now?

Comment (DaedalusGame) [reply src/main/java/teamroots/embers/recipe/RecipeRegistry.java]: Okay, I've added crafttweaker integration for steam engine fluids and boiler fluids now, just remove this change and the associated config option and i'll merge

Comment (Kurtchekov) [reply src/main/java/teamroots/embers/recipe/RecipeRegistry.java]: I've made the change you've asked for

**Pull request merged**